io_scene_dae/export_dae.py

```diff
@@ -1370,7 +1370,8 @@ def export_curve(self, curve):
         self.writel(
             S_GEOM, 1, "<geometry id=\"{}\" name=\"{}\">".format(
                 splineid, curve.name))
-        self.writel(S_GEOM, 2, "<spline closed=\"0\">")
+        self.writel(S_GEOM, 2, "<spline closed=\"{}\">".format(
+                'true' if curve.splines and curve.splines[0].use_cyclic_u else 'false'))
```

Comment:
Use double quotes for consistency with the rest of the codebase:

```suggestion
                "true" if curve.splines and curve.splines[0].use_cyclic_u else "false"))
```